go/consensus/tendermint/apps/staking/state/gas.go

```diff
@@ -24,6 +26,13 @@ type feeAccumulator struct {
 	balance quantity.Quantity
 }
 
+var MinTransactBalanceExemptMethods = map[transaction.MethodName]bool{
```

Comment:
This should be in `go/consensus/api/transaction` or similar.

Reply:
removing this from the pr, so gonna resolve this. let's keep it in mind if we later need exempt methods after all